File: src/main/kotlin/org/example/schwimmen/eingabe/Abwesenheiten.kt
package org.example.schwimmen.eingabe

fun parseAbwesenheiten(data: String): List<String> = parseAbwesenheiten(data.lines().map { it.split("\t") })

fun parseAbwesenheiten(rows: List<List<String>>): List<String> = rows.mapNotNull { it.getOrNull(0) }

File: src/ts/eingabe/abwesenheiten.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.parseAbwesenheiten = parseAbwesenheiten;
exports.parseAbwesenheitenFromGrid = parseAbwesenheitenFromGrid;
function parseAbwesenheiten(data) {
    return parseAbwesenheitenFromGrid(data.split("\n").map((row) => row.split("\t")));
}
function parseAbwesenheitenFromGrid(rows) {
    return rows.map((row) => row[0]).filter((it) => it != undefined && it.length != 0);
}
